xla/backends/gpu/runtime/dynamic_slice_thunk.h

```diff
@@ -47,8 +47,44 @@ class DynamicSliceThunk : public Thunk {
  public:
   // Dynamic slice offset can be either: (1) a statically known constant value
   // or (2) a truly dynamic offset that is computed on device and have to be
-  // transferred to host.
-  using Offset = std::variant<int64_t, BufferAllocation::Slice>;
+  // transferred to host or (3) a temporary HloModule that computes the offset
+  // with a single induction variable as the input.
+  using Offset = std::variant<int64_t, BufferAllocation::Slice, HloModule*>;
+
+  struct OffsetAsFunctionOfIndvarModulesMetadata {
+    // These two modules help keep track of the induction variable. The module
+    // `indvar_init_` is a module with prototype `() -> integer[]`. It takes
+    // no input, and returns the initial value of the induction variable. The
+    // module `indvar_update_` is a module with prototype `(integer[]) ->
+    // integer[]`. It takes the current value of the induction variable, and
+    // returns the next value of the induction variable.
+    std::unique_ptr<HloModule> indvar_init_, indvar_update_;
```

Comment:
According to the style guide, struct member variables don't need trailing underscores (unlike class member variables).  

That was flagged by linter.

Reply:
Should I address this here or is this already in the process of merge and changes now will cause issues?

Reply:
I see that you've changed it.
That's indeed what I was asking, sorry for not being clear!